estimator/serenity_estimator.cpp

```diff
@@ -35,14 +35,22 @@ class SerenityEstimatorProcess :
 
   Future<Resources> _oversubscribable(
       const Future<ResourceUsage>& _resourceUsage) {
-    Try<Resources> ret = this->pipeline->run(_resourceUsage.get());
+    // TODO(bplotka): Decide if need such simple algorithm.
+    Resources allocatedRevocable;
```

Comment:
Are you trying to avoid oversubscribing oversubscribed resources? We chatted about this before, let's chat about this with Christos and hear what they did. For example, it may be enough to wait until they get into a stable state and then oversubscribe again